backend/routes/profile.py

```
@@ -110,21 +111,21 @@
             detail="Invalid file type. Allowed: jpg, jpeg, png, gif, webp."
         )
 
-    secure_filename = f"{db_user.id}_{secrets.token_hex(8)}{file_extension}"
-    file_location = os.path.abspath(os.path.join(UPLOAD_DIR, secure_filename))
+    # ✅ Upload directly to Cloudinary
+    upload_result = upload_to_cloudinary(
+        file.file,  # sending the file object
+        folder_name="noobsquad/profile_pictures"
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)